public/src/app/sites.ts

```diff
@@ -8,4 +8,17 @@ export const sites = [
   's.keyman.com',
   'status.keyman.com', // ooh self referential!
   'developer.keyman.com'
-];
\ No newline at end of file
+];
+
+// TODO: this could be consolidated above
+export const siteSentryIds = {
```

Comment:
Isn't this more appropriate in sentry.component.ts?

Reply:
Yeah maybe ... or some centralised constants location given it overlaps.